app/controllers/application_controller.rb

```diff
@@ -11,10 +11,8 @@ class ApplicationController < ActionController::Base
 
   # whitelists attributes in devise
   def configure_permitted_parameters
-    [:name].each do |sym|
-      devise_parameter_sanitizer.for(:sign_up) << sym
-      devise_parameter_sanitizer.for(:account_update) << sym
-    end
+    devise_parameter_sanitizer.permit(:sign_up, keys: [:name])
+    devise_parameter_sanitizer.permit(:account_update, keys: [:name])
```

Comment:
I did this because it was mentioned as a change for devise 4.x (which we moved to from a previous version)